config/projects.yaml

```diff
@@ -295,20 +270,6 @@ media-bundle:
             variants:
                 symfony/symfony: ['4.4.*']
 
-notification-bundle:
```

Comment:
We should not remove this, because we have an active project that is still using this bundle:
https://github.com/sonata-project/SonataPageBundle/blob/3.x/composer.json#L36

Reply:
7 month ago was the last release, not sure if it is relevant to keep it here.

Reply:
And it won't block us to make new releases if needed (or even add the project back here)